data.sql

```diff
@@ -53,4 +53,32 @@ WHERE NAME IN ('Charmander', 'Squirtle', 'Blossom');
 UPDATE animals
 SET owner_id = 5
 WHERE NAME IN ('Angemon', 'Boarmon');
-SELECT * FROM animals;
\ No newline at end of file
+SELECT * FROM animals;
+
+INSERT INTO vets(name, age, date_of_graduation) VALUES
+('William Tatcher', 45, '2000-04-23'),
+('Maisy Smith', 26, '2019-01-17'),
+('Stephanie Mendez', 64, '1981-05-04'),
+('Jack Harkness', 38, '2008-06-08');
+
+INSERT INTO visits(animal_id, vet_id, date) VALUES
+((SELECT id FROM animals WHERE name = 'Agumon'),(SELECT id FROM vets WHERE name = 'William Tatcher'), '2020-05-24'),
+((SELECT id FROM animals WHERE name = 'Agumon'),(SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2020-07-22'),
+((SELECT id FROM animals WHERE name = 'Gabumon'),(SELECT id FROM vets WHERE name = 'Jack Harkness'), '2021-02-02'),
+((SELECT id FROM animals WHERE name = 'Pikachu'),(SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-05-01'),
```

Comment:
* Kindly insert the right date for this visit `Pikachu visited Maisy Smith on Jan 5th, 2020.` the `2020-05-01` is wrong